via/services/youtube_api/models.py

```diff
@@ -0,0 +1,219 @@
+import base64
+from copy import deepcopy
+from dataclasses import dataclass, field
+from itertools import zip_longest
+from operator import attrgetter
+from typing import List, Optional
+
+from via.services.youtube_api._nested_data import safe_get
+
+
+@dataclass
+class CaptionTrack:
+    """A source of transcription data, in a particular language."""
+
+    # Items which form the unique part
+    language_code: str
+    """Original language of the track."""
+
+    name: Optional[str] = None
+    """Human set name for the track."""
+
+    kind: str = None
```

Comment:
```suggestion
    kind: Optional[str] = None
```

Hazards of not running a type checker I guess